app/src/ui/app.tsx

```diff
@@ -1317,13 +1321,97 @@ export class App extends React.Component<IAppProps, IAppState> {
           />
         )
       case PopupType.MergeConflicts:
-        return (
-          <MergeConflictsWarning
-            dispatcher={this.props.dispatcher}
-            repository={popup.repository}
-            onDismissed={this.onPopupDismissed}
-          />
-        )
+        if (enableMergeConflictsDialog()) {
+          const selected = this.state.selectedState
+          if (selected === null || selected.type !== SelectionType.Repository) {
+            return null
+          }
+          const formState = selected.state.compareState.formState
```

Comment:
is this something we can add to `eslint` a la https://github.com/desktop/desktop/pull/5969?

Reply:
I was hoping it'd be as easy as that but sadly `yarn lint:fix` from that branch doesn't autofix. Happy to chat more about it over there, I just wanted to avoid this new code duplicating checks being overly verbose where possible.